.gitlab/packaging/oci.yml

```
@@ -75,6 +75,55 @@
     paths:
       - ${OMNIBUS_PACKAGE_DIR}
 
+.package_oci_win:
+  extends: .package_oci
+  script:
+    - set -euo pipefail
+    - PIPE_DIR="$OMNIBUS_PACKAGE_DIR/pipeline-$CI_PIPELINE_ID"
+    - ARTIFACT="$(ls -1 "${PIPE_DIR}"/${WIN_ARTIFACT_PATTERN} | head -n1)"
+    - '[ -n "${ARTIFACT}" ]'
```

Suggestion: rather than allowing several files to match the pattern and arbitrarily choose one, we should probably assert that there's really only a single file and otherwise fail, something like:

```suggestion
    - ARTIFACT="$(ls ${PIPE_DIR}"/${WIN_ARTIFACT_PATTERN})" || { echo "No suitable input artifact found"; exit 1; } 
    - [ "$(echo ${ARTIFACT} | wc -w)" = 1 ] || { echo "Expected to only one candidate input artifact"; exit 1; }
```

---

Made changes per suggestion